qa/tasks/cephfs/cephfs_test_case.py

```diff
@@ -206,7 +208,16 @@ def setUp(self):
 
         self.configs_set = set()
 
+        if self.last_active_mgr:
+            self.mon_manager.revive_mgr(self.last_active_mgr)
```

Comment:
Doesn't it suffice to fail the manager (done during teardown) and then wait a bit so that it's active and available for servicing requests. What's the need to revive the manager instance that was failed?